pyperformance/_venv.py

```diff
@@ -239,13 +239,14 @@ def upgrade_pip(self, *, installer=True):
         if ec != 0:
             raise RequirementsInstallationFailedError('pip')
 
-    def ensure_reqs(self, *reqs, upgrade=True):
+    def ensure_reqs(self, *reqs, upgrade=True, cwd=None):
         print("Installing requirements into the virtual environment %s" % self.root)
         ec, _, _ = _pip.install_requirements(
             *reqs,
             python=self.python,
             env=self._env,
             upgrade=upgrade,
+            cwd=cwd
```

Comment:
Presumably this is so pip will find the wheel file.  Something feels off to using cwd for this.

What are the alternatives?  What about:

* pass the resolved path to the wheel file as a requirement?
* use pip's `--index-url`, `--extra-index-url`, or `--find-links` args?

Reply:
Yeah -- `--find-links` seems like the better solution here. Let me poke at this and see if I can get it to work.